app/editor/src/features/content/form/constants/clipDirectoryColumns.tsx

```diff
@@ -44,44 +45,56 @@ export const clipDirectoryColumns = (
     id: 'size',
     Header: () => <div className="center">Size</div>,
     accessor: 'size',
-    Cell: ({ value }) => <div className="ft-row">{!!value ? `${value / 1000000} MB` : ''}</div>,
+    maxWidth: 35,
+    Cell: ({ value }) => (
+      <div className="ft-row">{!!value ? `${(value / 1000000).toFixed(2)} MB` : ''}</div>
+    ),
   },
   {
     id: 'modified',
     Header: () => <div className="center">Modified</div>,
     accessor: 'modified',
+    maxWidth: 40,
     Cell: ({ value }) => <div className="ft-row">{moment(value).format('DD-MM-yy hh:mm:ss')}</div>,
   },
   {
     id: 'actions',
     Header: () => <div>Actions</div>,
     accessor: 'isDirectory',
-    width: 80,
+    maxWidth: 30,
     Cell: ({ row, data }: any) => (
-      <div className={row.values.isDirectory ? 'hidden' : 'center'}>
-        <FaPlay className="stream" title="watch/listen/edit" onClick={() => onSelect(row.values)} />
-        <FaCloudDownloadAlt
-          className="download fa-lg"
-          title="download"
-          onClick={() => {
-            onDownload(row.values);
-          }}
-        />
-        <FaPaperclip
-          className="attach fa-lg"
-          title="Attach to snippet"
-          onClick={() => {
-            onAttach(row.values);
-          }}
-        />
-        <FaTrash
-          className="delete fa-lg"
-          title="Delete"
-          onClick={() => {
-            onDelete(row.values);
-          }}
-        />
-      </div>
+      <Row className={`file-actions ${row.values.isDirectory ? 'hidden' : 'center'}`} wrap="nowrap">
```

Comment:
Simplified the CSS.  I'd like to be able to remove the `fa-lg` classname values too at some point.